src/blockrelay/graphene.cpp

```diff
@@ -1182,12 +1182,17 @@ std::string CGrapheneBlockData::ReRequestedTxToString()
 bool CGrapheneBlockData::CheckGrapheneBlockTimer(const uint256 &hash)
 {
     // Base time used to calculate the random timeout value.
-    static int64_t nTimeToWait = 1000;
+    static uint64_t nTimeToWait = GetArg("-graphene-timer", DEFAULT_GRAPHENE_TIMER);
+    if (nTimeToWait == 0)
+    {
+        LOG(GRAPHENE, "Graphene preferential times is disabled\n");
+        return false;
```

Comment:
I would however take these log entries out...otherwise there can be quite a few for each block.  If the timer is enabled you'll see it as a single entry which is handled below.